### PR TITLE
feat: add "orders" subcommand

### PR DESCRIPTION
**Pull request by divy-work**

### CODE REVIEW

File: src/commands/orders.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { init, collectDatabase, TokenInstance } from "@utils/database";
+import { asTree } from "@utils/tree";
+
+export default async () => {
+  const connection = await init("./db.db");
```
Comment (johnletey) [inline src/commands/orders.ts:5]: @divy-work What if the user is using a custom db file? 😄

Comment (johnletey) [reply src/commands/orders.ts]: Resolved in https://github.com/useverto/trading-post/pull/32/commits/4c980beec7812242f8ffe0a73582326b73a1a389

Comment (divy-work) [reply src/commands/orders.ts]: Oh was just pushing that :laughing:

**Review by johnletey — APPROVED** [review]
Great work! 😍

**Pull request merged**